quickfixj-codegenerator/src/main/java/org/quickfixj/codegenerator/MessageCodeGenerator.java

```diff
@@ -134,6 +142,19 @@ private void generateFieldClasses(Task task) throws ParserConfigurationException
                     Map<String, String> parameters = new HashMap<>();
                     parameters.put("fieldName", fieldName);
                     parameters.put("fieldPackage", task.getFieldPackage());
+                    String utcTimestampPrecision = task.getUtcTimestampPrecision();
+                    if (utcTimestampPrecision != null) {
+                        String utcTimestampPrecisionParameterName = "utcTimestampPrecision";
+                        if (!UTC_TIMESTAMP_PRECISION_ALLOWED_VALUES.contains(utcTimestampPrecision)) {
+                            throw new CodeGenerationException(new IllegalArgumentException(String.format(
+                                "Allowed values for parameter %s is %s. Supplied value: \"%s\".",
```

Comment:
Trivial: I think it should be "are".

Reply:
Fixed in ce5f595346c622671dc741ea09c6242ef5a7e81b.